controllers/app.js

```diff
@@ -1,6 +1,9 @@
 const app = angular.module('InvertedIndexApp', []);
 
 app.controller('InvertedIndexController', ($scope) => {
+
+  $scope.allValidatedBooks = {};
+
   const numToArray = (n) => {
```

Comment:
**Critical**
avoid using keywords like Array, Set, Object, Number et.c. in your naming conventions